GameData/RealismOverhaul/Engine_Configs/RD100_Config.cfg

```diff
@@ -363,8 +368,8 @@
 
 			atmosphereCurve
 			{
-				key = 0 248
-				key = 1 220
+				key = 0 300
```

Comment:
300 seconds?

Reply:
> Designation   Thrust, kN       ...       Specifc impulse, s
> ...
> RD-103M       432(SL)/500(V)        323(SL)/300(V)

So, yeah 300 seconds for the 103M is what it says.